catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/SortedQueryMonitor.java

```diff
@@ -163,8 +163,7 @@ public void run() {
                         Exceptions.getFullMessage(e),
                         e);
 
-                processingDetails.add(new ProcessingDetailsImpl(sourceId,
-                        new Exception(Exceptions.getFullMessage(e))));
+                processingDetails.add(new ProcessingDetailsImpl(sourceId, e));
```

Comment:
@lessarderic Already confirmed with @stustison regarding these changes. We shouldn't be instantiating an instance of `Exception` and passing it to the `ProcessingDetails` because then my `generateSanitizedMessages` method will return the unhelpful `Internal error` message where we might have instead desired `Query invalid` or `Query hit a source that was not available` since I'm doing checks against the `instanceof` for each exception.